backend/init_db.py

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+"""
+Database initialization script.
+"""
+
+from app.core.database import create_db_and_tables
+from app.core.settings import get_settings
+
+
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)